src/index.js

```diff
@@ -1 +1,35 @@
 import './css/index.css';
+import displayLists from './modules/display';
+
+const nameForm = document.querySelector('#name');
+const scoreForm = document.querySelector('#score');
+const submitBtn = document.querySelector('.btn');
+const gameId = 'eU8fUe323J6Z3a6jqAvT';
+
+const getScore = async () => {
+  const response = await fetch(
+    `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores/`,
+    {
+      method: 'POST',
+      body: JSON.stringify({
+        user: nameForm.value,
+        score: scoreForm.value,
+      }),
+      headers: {
+        'Content-type': 'application/json',
+      },
+    },
+  );
+  await response.json();
+  nameForm.value = '';
+  scoreForm.value = '';
+};
+submitBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  getScore();
+  const error = document.querySelector('.error');
+  if (!nameForm && !scoreForm) {
+    error.innerHTML = 'Please fill all input fields';
+  }
+});
+document.addEventListener('DOMContentLoaded', displayLists);
```

Comment:
- [ ] [OPTIONAL] The way you are using this function `getScore`, should reflect in its name to avoid misleading anyone reading your code. getScoe sound like fetchScore, a get request, but it is rather making a post request. I see that this function actually Posts the name and score. Its name should be **_setScore_** or **_postScore_**.

Reply:
Sure, I will work on it. Thanks